crowbar_framework/lib/crowbar/validator/network_validator.rb

```diff
@@ -4,7 +4,7 @@ class NetworkValidator
       def validate_network
         `#{Rails.root.join("..", "bin/network-json-validator").to_s} \
           --admin-ip \
-          #{Socket.ip_address_list.detect(&:ipv4_private?).ip_address} \
+          #{IPSocket.getaddress(Socket.gethostname)} \
```

Comment:
Testing this locally, if the hostname is missing, how it will try to resolve it? Because I dont have my hostname on /etc/hosts and it still resolves it to my VPN IP :D